utils/src/test/scala/za/co/absa/enceladus/utils/types/TypedStructFieldSuite.scala

```diff
@@ -15,21 +15,24 @@ class TypedStructFieldSuite extends FunSuite {
   private def createField(dataType: DataType,
                           nullable: Boolean = false,
                           default: Option[Any] = None,
-                          otherMetadata: Map[String, String] = Map.empty
+                          otherMetadata: Map[String, Any] = Map.empty
```

Comment:
This guy is missing the copyright

Reply:
Weird. Sometimes my template doesn't kick in, seems. Thanks for noticing, will add it.